src/Sarif.Viewer.VisualStudio.Core/OpenLogFileCommands.cs

```diff
@@ -259,7 +259,7 @@ private async System.Threading.Tasks.Task MenuItemCallbackAsync(object sender, E
             try
             {
                 await ErrorListService.ProcessLogFileAsync(logFile, toolFormat, promptOnLogConversions: true, cleanErrors: true, openInEditor: true).ConfigureAwait(continueOnCapturedContext: false);
-                new DataService().CloseEnhancedResultData();
+                new DataService().CloseEnhancedResultData(0);
```

Comment:
>0 [](http://example.com/codeflow?start=58&length=1)

suggest using a named parameter here, 'cookie: 0'

Reply:
applied this change in #535